app/views/admin/merchants/new.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Create a New Merchant</h1>
+
+<%= form_with url: "/admin/merchants", method: :post, local: true do |f| %>
```

Comment:
You decided to go the form_with url vs model route after all?

Reply:
Yeah I mentioned in the PR that I started with model but had some weird routing thing going on and needed to change it to url to get it working for now.  I'll go back and work with it some more